src/Redux.elm

```diff
@@ -24,25 +25,20 @@ view model =
 program :
     { init : ( model, Cmd msg )
     , update : msg -> model -> ( model, Cmd msg )
+    , encode : model -> Json.Value
```

Comment:
I see. It turns out that Elm automatically decodes the json values. So, the JS world will get an object even though we are sending json through the port.